wheatley/tower.py

```diff
@@ -69,11 +66,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self._socket_io_client = None
 
     @property
-    def number_of_bells(self):
+    def number_of_bells(self) -> int:
         """ Returns the number of bells currently in the tower. """
         return len(self._bell_state)
 
-    def ring_bell(self, bell: Bell, is_handstroke: bool):
+    def ring_bell(self, bell: Bell, is_handstroke: Stroke) -> bool:
         """ Send a request to the the server if the bell can be rung on the given stroke. """
         try:
             stroke = self.get_stroke(bell)
```

Comment:
This needs to be a bool to send to RingingRoom.
It also errors locally
> ERROR:TOWER:Object of type Stroke is not JSON serializable

Reply:
I'm struggling to make a suggestion, but feel we should rename `is_handstroke` and can do better than `stroke` for the current state of the bell

Reply:
To be honest, I think that if the only bell state is a stroke (and it has type `Stroke`), then I don't see any issue with calling the variable `expected_stroke` - it feels pretty succinct and doesn't clash with the other local variable `stroke`.